### ISSUE TITLE
After loading the `show` view for a ticket, navigating 'back' to the index does not load more tickets

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#3466]
If there is one ticket already loaded from navigating directly to an individual ticket, then the system seems to hang when you use the `back` button to go to the ticket index. Refreshing the index will cause the tickets to load.

**Comment by bwreid** [comment]
I'm not seeing this behavior locally or on production. When I go directly to a ticket page, then hit the `<- Ticketing Dashboard` link, I am brought back to the index with the default "Owners" of "unassigned" and my username. This shows no tickets for me, but if I change the filters, everything seems to show up. Is there a more detailed way to access this error?

**Comment by ragesoss** [comment]
Hmm... maybe it's only if you reply and resolve it? I'm pretty sure I've had this happen (in addition to the Wikipedia Experts) but I also can't replicate it with the back link.

**Comment by ragesoss** [comment]
Nope, that's not it either. Guess we need to gather more info about how to replicate it.
